addons/mrp/static/src/js/mrp_bom_report.js

```diff
@@ -242,6 +249,30 @@ var MrpBomReport = stock_report_generic.extend({
             $parent.remove();
         });
     },
+    _unfoldedIDs: function() {
+        return _.map(this.$el.find('.o_mrp_bom_foldable').closest('tr'), function (el) {
+            return $(el).data('id');
+        });
+    },
+    _unfold: function (ids) {
+        var self = this;
```

Comment:
No need
```suggestion
```
I guess it's because there is a `self` in line 272 but it can be `this`, that makes no difference.
It's useful to save `this` into a variable when the reference of it is lost. Typically, it happens inside a function declared in the old way (`function () {...}`).
One of the purpose of using arrow function (`() => {...}`) is `this` still refer to the same object.